widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java

```diff
@@ -22,11 +22,14 @@ public class ChatActivity extends AppCompatActivity {
     private ChatView.OnBackClickedListener onBackClickedListener = () -> {
         if (chatView.backPressed()) finish();
     };
+
+    // TODO deprecate UiTheme pass GliaSdkConfiguration instead (remove global sdk configuration holder in ChatActivity)
```

Comment:
Will you process this TODO?

Reply:
No, its for future development (we promised to leave suggestions that we came up with that could or should be implemented in the future - this change will require deprecation notice)

Reply:
Could create a technical task out of it? (label suggestion?) cc: @dukhovnyi

Reply:
My point of view is:
- any label that looks clear to you and describes the scope of work
- either leave this TODO or add a reference to the code in the task